### PR TITLE
non gmcp skill recognition for install and steam pipe fix.

### PR DESCRIPTION
**Pull request by steingrim**
enables skill recognition for installdata of the install script.

Constains a fix steam pipe to display correctly under MMCONFIG2.

Does **not** fix parry or stancing.

### CODE REVIEW

**Comment by steingrim** [comment]
Just checking to make sure this didn't fall though the cracks.

File: raw-mm.aliases.lua [hunk 1]
```diff
@@ -231,66 +231,19 @@ end
...
+  cecho("<a_darkblue>--<purple>(m&mf) <a_grey>Configuration, MMCONFIG2 continued<a_darkblue>" .. string.rep("-", 48) .. "\n")
+
+  local c1,s1 =
+  unpack(pipes.steam.arty and
+      {"<gold>", "Arty"} or
+          (pipes.steam.lit and {"<a_yellow>", "Lit!"} or {"<a_darkgrey>", "Unlit."})
+  )
+ 
+  -- I know this is ugly but the 'f stringformated hated the versions of it I tried. This works and it can be made pretty some other day.'
+  cecho(f("\n<a_darkcyan>  Steam Pipe: <a_grey>ID " ..pipes.steam.id .."   puffs: " ..pipes.steam.puffs .."   lit? " .. c1 ..s1))
```
Comment (vadi2) [inline raw-mm.aliases.lua]: What issue were you having with this? We should be able to make it work.

File: raw-mm.install.lua [hunk 2]
```diff
@@ -972,27 +996,29 @@ function installstart(fresh)
...
     end
-  else
-    for skill, skilldata in pairs(installdata) do
-      if skilldata.gmcp then
-        sendGMCP("Char.Skills.Get "..yajl.to_string(skilldata.gmcp))
-        ids[skill] = true
-      end
-    end
+--   else
+--     for skill, skilldata in pairs(installdata) do
```
Comment (vadi2) [inline raw-mm.install.lua]: Let's delete it for now, can always pull from git history later.

File: raw-mm.aliases.lua (hunk 1 above)
Comment (steingrim) [reply raw-mm.aliases.lua]: The issue was that using 'f' to format strings is more particular than using '..var'. and 'f' was choking on out of scope or it was just insisting that 'pipes' was a global.

refactored the code and made a local copy or pipes and sent that.

Below you'll see some of the syntax I tried. It is there just for your eyes and if you wanted to remove it to make the commit cleaner please do.

File: raw-mm.install.lua (hunk 2 above)
Comment (steingrim) [reply raw-mm.install.lua]: done

**Review by steingrim — COMMENTED** [review]
can be deleted or squished.

only here in case it reveals some behavior that you feel is a bug. like why the var would work with .. and not 'f'.

'f'string format Failure recap 1. This version doesn't work.
3b2477c

'f'string format Failure recap 2. This version doesn't work.
8f0fb72

**Review by vadi2 — APPROVED** [review]

**Pull request merged**